addons/binding/org.openhab.binding.boschindego/build.properties

```diff
@@ -0,0 +1,8 @@
+source.. = src/main/java/
+output.. = target/classes
+bin.includes = META-INF/,\
+               .,\
+               OSGI-INF/,\
+               ESH-INF/,\
+               lib/bosch-indego-controller-lib-0.8-jar-with-dependencies.jar
```

Comment:
This file is pretty big and contains a lot that is anyhow already available (all Apache Commons stuff).
In good OSGi manner, you should rather use the library without its dependencies and resolve the rest from OSGi bundles.